lib/karto_sdk/include/karto_sdk/Karto.h

```diff
@@ -4338,9 +4362,10 @@ namespace karto
      */
     void Update()
     {
+      int laserResidual = !GetIs360Laser();
```

Comment:
You find if its 360 and give it a bool, the variable is implicitly stored as a bool, the `Get` is a double, and you're assigning it an int. This needs to be cleaned up.